src/Pos/GState/BlockExtra.hs

```diff
@@ -169,6 +169,7 @@ initGStateBlockExtra :: MonadDB m => HeaderHash -> m ()
 initGStateBlockExtra firstGenesisHash = do
     gsPutBi (mainChainKey firstGenesisHash) ()
     gsPutBi (forwardLinkKey genesisHash) firstGenesisHash
+    gsPutBi lastSlotsKey noLastBlkSlots
```

Comment:
With this change I would also modify `getLastSlots` to throw `DBMalformed` if this key is not in DB instead of returning `noLastBlkSlots`.